content/tracing/setup/php.md

```diff
@@ -8,75 +8,100 @@ further_reading:
 - link: "tracing/visualization/"
   tag: "Documentation"
   text: "Explore your services, resources and traces"
-- link: "https://goo.gl/forms/rKjH2J6nJ585KXri2"
-  tag: "Survey"
-  text: Request Beta Access
+- link: "tracing/advanced_usage/?tab=php"
+  tag: "Advanced Usage"
+  text: "Advanced Usage"
 ---
 
 <div class="alert alert-warning">
-The APM tracer for PHP applications is in alpha and not officially supported by Datadog. The officially supported beta will launch as early as October 2018. We do not recommend running the alpha tracer in production.
+The APM tracer for PHP applications is in Open Public Beta.
 </div>
 
-{{< whatsnext desc="To be notified of the beta launch, submit this form:">}}
-    {{< nextlink href="https://goo.gl/forms/rKjH2J6nJ585KXri2" tag="Survey" >}}PHP Beta Access Survey{{< /nextlink >}}
-{{< /whatsnext >}}
+## Installation and Getting Started
 
-<br>
+To begin tracing applications written in PHP, first [install and configure the Datadog Agent][1] (see additional documentation for [tracing Docker applications][7]).
 
-The unstable alpha tracer can be [accessed today on Github][2].
+Next, install [from source][3] the Datadog PHP extension or by using one of the precompiled packages for supported distributions. Latest release packages can be found on GitHub's [dd-trace-php releases page][4]:
+
+```bash
+# using RPM package (RHEL/Centos 6+, Fedora 20+)
+rpm -ivh ddtrace.rpm
+
+# using DEB package (Debian Jessie+ , Ubuntu 14.04+)
+deb -i ddtrace.deb
+
+# using tar.gz archive (Other distributions using libc6)
+tar -xf ddtrace.tar.gz -C /
+/opt/datadog-php/bin/post_install.sh
+```
+
+Finally, install DataDog tracing library using composer:
```

Comment:
```suggestion
Finally, install the DataDog tracing library using composer:
```

Reply:
Also: Datadog, not DataDog.

Reply:
Ah - my bad - Probably too much PHP CamelCase